src/containers/PageConstructor/PageConstructor.tsx

```diff
@@ -99,7 +99,7 @@ export const Constructor = (props: PageConstructorProps) => {
         <InnerContext.Provider value={context}>
             <RootCn>
                 <div className={b('wrapper')}>
-                    {themedBackground && (
+                    {Object.keys(themedBackground).length && (
```

Comment:
there would be an error if we don't have `themedBackground` value

Reply:
I've checked, if background = {} as a default, then themedBackground also {}

Reply:
> I've checked, if background = {} as a default, then themedBackground is also {}

why can't we just delete it's default value?